### PR TITLE
Display add album

### PR DESCRIPTION
**Pull request by eamador1**
In this branch, the following has been done:
---

- Add music albums
- Display music albums

### CODE REVIEW

File: create_instances_module.rb [hunk 1]
```diff
@@ -43,6 +44,19 @@ def add_game
...
   end
 
+  def add_music_album
+    p 'Is the album in Spotify (true or false)?'
+    spotify = gets.chomp
+
+    p 'Add publish date (DD-MM-YYYY):'
+    album_publish_date = valid_date_input
+
+    base_album = MusicAlbum.new(album_publish_date, spotify)
```
Comment (Estete9) [inline create_instances_module.rb:54]: Great job with this method my friend, it's almost perfect.
If you look closely at your `MusicAlbum` class you can notice that first, you ask for `spotify` and then the publish date.

Please change the position of what you are passing to the `MusicAlbum.new()` constructor (ex: `MusicAlbum.new(spotify, album_publish_date)`)

**Review by Estete9 — CHANGES_REQUESTED** [review]
Hi @eamador1 
I'll review this PR. ✋

Good job so far it looks almost perfect!



There are some recommendations I would like you to take into account. Please read the inline comments and the the required changes in this description

Highlights
 You have written a clean code ✅
 The distribution, padding, and margins are perfect. ✅
 Good PR title/description ✅
Required Changes ♻️
Please add the functionality to list all the `genres` as per the ticket requirements

image
<br>
<br>
 Also please check the inline comments under the code.

Cheers and Happy coding!👏👏👏

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @Estete9 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/Estete9">
   @Estete9
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/esteban-palacios-5030a772/">
   linkedin-esteban-palacios-5030a772
</a>

</td>
<td>

<a href="https://twitter.com/NaughTban">
   @NaughTban
</a>

</td>
</tr>

**Comment by eamador1** [comment]
Since the last review, the following has been done:
---

- List all genres
- Shorten display module
- reorder arguments

**Review by joramot — APPROVED** [review]
Hi @eamador1  ,
Your feature is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

STATUS: APPROVED 🟢

Highlights
Good PR title ✅
Descriptive PR description ✅
Implement the features ✅
Great use of modules ✅
Linters are green ✅
Meaningful Github commits ✅

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

⚠️ WARNING ⚠️
Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Pull request merged**